lib/drivers/animasDriver.js

```diff
@@ -1601,16 +1599,16 @@ module.exports = function (config) {
     var wizard = null;
     for(var b in data.wizardRecords) {
       var wizarddatum = data.wizardRecords[b];
-      var bolusdatum = postrecords[b];
+      var bolusdatum = _.findWhere(postrecords, { 'index': wizarddatum.index, 'syncCounter': wizarddatum.sync_counter });
```

Comment:
I think it's best to stick to `_.find` in general. Same args...the only difference is `_.findWhere` will do a deep comparison where applicable, which is less efficient (and we had a problem once with this in blip *ages* ago, so I have a knee-jerk "avoid lodash `*where` methods!" reaction).